tests/integration/component.go

```diff
@@ -835,22 +835,21 @@ func componentTests(args ...string) {
 			helper.CopyExample(filepath.Join("source", "nodejs"), context)
 			helper.CmdShouldPass("odo", append(args, "create", "nodejs", cmpName, "--app", appName, "--project", project, "--context", context)...)
 			helper.CmdShouldPass("odo", "url", "create", "example", "--context", context)
-			helper.CmdShouldPass("odo", "storage", "create", "storage-name", "--size", "1Gi", "--path", "/data", "--context", context)
+
+			// TODO: Add this When the issue #2710 gets fixed
+			// helper.CmdShouldPass("odo", "storage", "create", "storage-name", "--size", "1Gi", "--path", "/data", "--context", context)
 			helper.ValidateLocalCmpExist(context, "Type,nodejs", "Name,"+cmpName, "Application,"+appName, "URL,0,Name,example")
 			helper.CmdShouldPass("odo", append(args, "push", "--context", context)...)
 
-			helper.CmdShouldPass("odo", "url", "create", "example-1", "--context", context)
-			helper.CmdShouldPass("odo", "storage", "create", "storage-name-1", "--size", "1Gi", "--path", "/data-1", "--context", context)
-			helper.CmdShouldPass("odo", append(args, "push", "--context", context)...)
+			// delete with --wait flag
```

Comment:
@adisky Deleting these lines may cause some real functionality validation miss. These lines were added due to some reason may be to catch any regression for specific scenario. IMO create a separate spec to verify your changes.

Reply:
@amitkrout this test was not catching any regression, This line would always be true.  https://github.com/openshift/odo/blob/master/tests/helper/helper_oc.go#L458
I have fixed the test case and removed not necessary checks. Please check the regression it missed
https://github.com/openshift/odo/issues/2710
https://github.com/openshift/odo/issues/2693

Reply:
you can run the test case manually in verbose mode, and verify it is passing even if the pvc still exist, however it should fail here https://github.com/openshift/odo/blob/master/tests/integration/component.go#L846

Reply:
@adisky Understood, but tbh you are destroying one existing scenario with your changes. Why i am saying because the scenario is
```
create component 
create url
Create storage
odo push
create url
create storage
odo push
normal delete component 
```
And you are skipping steps of url create, storage create and push. IIRC the steps of running url create,storage create and push twice was to catch some short failure. So don't skip those steps. Rest script changes looks good to me.

Reply:
@amitkrout sure i would add it, but we cannot add storage as of now, due to  #2710